server/test/services/export/PdfExporterTest.java

```diff
@@ -227,7 +227,7 @@ public void statusTrackingDisabled() throws IOException, DocumentException {
           + "-- my-file-key UPLOADED (click to download) --\n"
           + "Answered on : 1969-12-31\n"
           + "applicant household members\n"
-          + "item1\n"
+          + "item1;\n"
```

Comment:
I don't think we want ; here as there's visual newline separators already

Reply:
\n isn't a newline in HTML, it's just a space. Any series of whitespace characters in HTML is treated as one space. So Admins don't know when one name ends and another name begins.

Reply:
This is the PDF test though.  I assume in the PDF the newlines are rendered as newlines; such that now they will have semicolons after them

Reply:
have changed only the UI now. Replaced "\n" with "; "